__init__.py

```diff
@@ -104,7 +104,7 @@ def run(self):
                 key = getNameKey(cardData['name'])
                 cardData['key'] = key
                 self.finalDataQueue.put(cardData)
-                self.imageQueue.put((cardData['key'], cardData['imageUrl']))
+                self.imageQueue.put((cardData['key'], cardData['variations'][0]['art']['fullsizeImageUrl']))
```

Comment:
I've introduced the notion of variations, of which each can have their own artwork.

Gwentify will only ever have one variation, so it's almost redundant here, but it is part of our final goal.